resource-managers/kubernetes/core/src/test/scala/org/apache/spark/deploy/k8s/KubernetesConfSuite.scala

```diff
@@ -55,7 +55,8 @@ class KubernetesConfSuite extends SparkFunSuite {
       APP_ID,
       None,
       MAIN_CLASS,
-      APP_ARGS)
+      APP_ARGS,
+      None)
```

Comment:
Commonly, to improve readability, we assign names to the the None params (e.g. foobaz=None) rather than positional arguments.